compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToNVVM.cpp

```diff
@@ -145,7 +148,7 @@ struct ConvertToNVVMPass : public ConvertToNVVMBase<ConvertToNVVMPass> {
       populateNVGPUToNVVMConversionPatterns(converter, llvmPatterns);
       populateGpuWMMAToNVVMConversionPatterns(converter, llvmPatterns);
       LLVMConversionTarget target(getContext());
-      configureGpuToNVVMConversionLegality(target);
```

Comment:
THis might be too disruptive of a change. Can we inline this method, i.e. all of 
https://github.com/llvm/llvm-project/blob/30b5d4a76357feebf4797d1d80bc9d5608c74a88/mlir/lib/Conversion/GPUToNVVM/LowerGpuOpsToNVVMOps.cpp#L307 and just drop the illegal ops that are causing a failure (the fma op being illegal AFAIK).

Reply:
Done, in fact I just kept the call and then added a `addLegalOp` to undo the `addIllegalOp`.

Reply:
ah... I am not sure that is a good idea.. I dont know how the tracking of legal/illegal op goes and its preferences and how much gaurantees that gives. I think it is better to just not call the upstream method. Copy it here and drop the fma from illegal op?

Reply:
I don't know the expected behavior either.. it could be undefined behavior

Reply:
I just changed it. If it passes, good to land from me.